tubesync/sync/management/commands/update-media-paths.py

```diff
@@ -0,0 +1,82 @@
+import os
+from django.core.management.base import BaseCommand
+from common.logger import log
+from sync.models import Source, Media
+import glob
+import shutil
+
+
+class Command(BaseCommand):
+
+    help = 'Updates media when the media format has changed naming/directory'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--source",
+            help="Only updates specified source"
+        )
+
+        parser.add_argument(
+            "--no-rename",
+            action="store_true",
+            help= "Do not rename the file (only move)"
+        )
+
+        parser.add_argument(
+            "--dryrun",
+            action="store_true", #treats as boolean
+            help="Don't make any changes, only print outputs"
+        )
+
+    def handle(self, *args, **options):
+        filtered_source = options['source']
+        dry_run = options['dryrun']
+        no_rename = options['no_rename']
+        if dry_run:
+            log.info('Dry run initiated')
+
+        if filtered_source:
+            log.info(f'Searching for source: {filtered_source}')
+            sources = Source.objects.filter(name=filtered_source)
```

Comment:
```suggestion
            sources = Source.objects.filter(name__contains=filtered_source)
```

I don't want to remember the full name and pass it to your tool.

Also, when doing this from the command line I tend to use `directory` because that is what I see on the filesystem.